src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1396,7 +1394,7 @@ MessageHelper::CreateGlobalPropertiesRequestsToHMI(
 
   // TTS global properties
   if (can_send_vr && (app->help_prompt() || app->timeout_prompt())) {
-    correlation_id = app_mngr.GetNextHMICorrelationID();
+    uint32_t correlation_id = app_mngr.GetNextHMICorrelationID();
```

Comment:
So previously we have two calls of GetNextHMICorrelationID(), now we have one, so this changes are valid? Please check regression related to this calls

Reply:
GetNextHMICorrelationID() returns next available correlation id(using counter). I don't see problem here. But of course regression tests will be executed.